src/domain/alerts/alerts.repository.ts

```diff
@@ -100,7 +101,14 @@ export class AlertsRepository implements IAlertsRepository {
         newSafeState,
       });
     } catch {
-      await this._notifyUnknownTransaction({ chainId, safeAddress, emails });
+      const emails = verifiedAccounts.map(
+        (account) => account.emailAddress.value,
+      );
+      await this._notifyUnknownTransaction({
+        chainId,
+        safeAddress,
+        emails,
+      });
```

Comment:
It's out of the scop eof this PR, but we will need to send the unsubscribe token here eventually.